types/src/resources/currency_info.rs

```diff
@@ -0,0 +1,141 @@
+use crate::{
```

Comment:
Shouldn't we add a copyright notice at the top of the file as in `src/types/event.rs`?

Reply:
Done